src/ducks/components/intents/InstallAppIntent.jsx

```diff
@@ -161,6 +161,7 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
 // translate last to pass the lang property to fetchApps()
 export default compose(
   translate(),
+  withClient,
```

Comment:
We don't need the client elsewhere in this component? Why do we need the `withClient` here then?

Reply:
Ok I think this was a duplication, in the changes I see withClient twice

Reply:
Spot on, thx.